definitions/schedule.py

```diff
@@ -35,36 +35,70 @@
                                      "united-kingdom-deaths",
                                      "united-kingdom-admissions"]),
 	        Dataset("united-kingdom-full",
-                    data_dir="subnational/united-kingdom/cases"),
+                    data_dir="subnational/united-kingdom/cases",
```

Comment:
Sorry to have missed this on the first review - the data dir should include -full on the second part (https://github.com/epiforecasts/covid-rt-estimates/blob/master/R/lists/dataset-list.R#L231 )
e.g. subnational/united-kingdom-full/cases or region/cases-full